recipes_source/recipes_index.rst

```diff
@@ -34,7 +34,7 @@ Recipes are bite-sized, actionable examples of how to use specific PyTorch featu
    :header: Loading data in PyTorch
    :card_description: Learn how to use PyTorch packages to prepare and load common datasets for your model.
    :image: ../_static/img/thumbnails/cropped/loading-data.PNG
-   :link: ../recipes/recipes/loading_data_recipe.html
+   :link: ../beginner/basics/data_tutorial.html
```

Comment:
since we are removing it, just delete line 37.

Reply:
I thought the ask was to deprecating and **creating a redirect from this tutorial to the data_tutorial.html**. Wouldn't this remove the tutorial entirely without a redirect?

Reply:
No, it will still convert that .rst to html but, you can add [:orphan:](https://www.sphinx-doc.org/en/master/usage/restructuredtext/field-lists.html#metadata) at the top of the .rst page so Sphinx doesn't drop a warning in the log. This whole card needs to go